### DISCUSSION TITLE
Does the library work with Nextjs 14 app router?

### DISCUSSION THREAD

**Discussion by mkotamies** [caching-tools/next-shared-cache#378] [Q&A] [answered]
Documentation doesn't separate app router and pages router configuration. Does the router matter and is the library compatible with Nextjs 14 app router?

**Comment by better-salmon** [comment, marked as answer]
Hello @mkotamies! The library is almost fully compatible with the App Router, with only one limitation. Currently, `revalidatePath` from `next/cache` is not functioning. However, I am actively working on a solution and hope to have it resolved in the upcoming minor update.

**Reply by markedwards** [reply]
Is this still the case?  Is this resolved?

**Reply by better-salmon** [reply]
@markedwards hi! Thank you for your question! `revalidatePath` is now working in the App Router since version 1.1.0.
